tuna/go_fish.py

```diff
@@ -565,8 +492,6 @@ def compose_worker_list(res, args):
       LOGGER.info('launch mid %u, proc %u', machine.id, gpu_idx)
       if not launch_worker(gpu_idx, f_vals, worker_lst, args):
         break
-      if args.init_session:
```

Comment:
this is a short cut to only run init_session once, otherwise its ran as many times as there are GPU...which is fine, there is a duplicate check in the DB to avoid this, but its better if we stop it here. On a system with 8 gpus you would see the duplicate err 7 times.

Reply:
launch_worker returns false when init_session is true, so this is no longer necessary.